serverless.test.yml

```diff
@@ -1,6 +1,380 @@
-- name: responds with 200 and Yeet
+- name: default responds with 200 and Yeet
   endpoint: {function: hello}
   response:
     status: 200
     body: 
-      message: "Yeet!"
\ No newline at end of file
+      message: 'Yeet!'
+
+- name: user create returns 200
+  endpoint:
+    method: POST
+    path: users/create
+  request:
+    body:
+      id: 144444
+      fname: test
+      lname: dude
+      faculty: Science
+  response:
+    status: 200
+    body:
+      message: 'Created!'
+
+- name: user create no ID returns 406
+  endpoint:
+    method: POST
+    path: users/create
+  request:
+    body:
+      fname: foo
+      lname: bar
+      email: foo
+      faculty: bar
+  response:
+    status: 406
+    body:
+      message: "User ID not specified."
+      data:
+        fname: foo
+        lname: bar
+        email: foo
+        faculty: bar
+
+- name: user get no ID returns 406
+  endpoint:
+    method: GET
+    path: users/get
+  response:
+    status: 406
+    body:
+      message: "User ID not specified."
+      data: null
+
+- name: user get found returns 200
+  endpoint:
+    method: GET
+    path: users/get
+  request:
+    form:
+      id: 144444
+  response:
+    status: 200
+    body:
+      id: 144444
+      fname: test
+      lname: dude
+      faculty: Science
+
+- name: user get not found returns 404
+  endpoint:
+    method: GET
+    path: users/get
+  request:
+    form:
+      id: 0
+  response:
+    status: 404
+    body: "\"User not found.\""
```

Comment:
is this shorthand for 
```
body:
    message: 
```
?

Reply:
Nah it's just a body, `\"` so that it matches the JSON string returned that has quotes